helm_deploy/laa-crown-court-contribution/templates/_environment.tpl

```diff
@@ -48,6 +40,26 @@ env:
       secretKeyRef:
         name: rds-postgresql-instance-output
         key: database_password
+  - name: MAAT_API_OAUTH_CLIENT_ID
+    valueFrom:
+        secretKeyRef:
+            name: hardship-api-oauth-client-id
+            key: MAAT_API_OAUTH_CLIENT_ID
+  - name: MAAT_API_OAUTH_CLIENT_SECRET
+    valueFrom:
+        secretKeyRef:
+            name: hardship-api-oauth-client-secret
+            key: MAAT_API_OAUTH_CLIENT_SECRET
+ - name: HARDSHIP_API_OAUTH_CLIENT_ID
+    valueFrom:
```

Comment:
The indentation is incorrect here. Idea allows you to render Helm templates, which should help you spot issues like this before pushing.